api/tests/pokemon.test.js

```diff
@@ -47,5 +48,16 @@ describe('GET /pokemon', () => {
   it('should accept a limit as an integer', async () => {
     const pokemon = await Posts.getPokemans(path, '40')
     expect(pokemon.body.results.length).toBe(40)
+  }) 
+  // Tried to test the "Count" key's value ->
+  it('Check the count', async () => {
```

Comment:
let's follow the same patterns as the other test descriptions and be more clear with our naming -- e.g: `it('should have a total pokemon count of 1154')`